test/reconciler-tests.sh

```diff
@@ -48,8 +48,14 @@ go_test_e2e -tags=e2e,cloudevents -timeout=1h ./test/e2e_new_channel/... || fail
 
 go_test_e2e -tags=deletecm ./test/e2e_new/... || fail_test "E2E (new deletecm) suite failed"
 
+echo "Running E2E Reconciler Tests with strict transport encryption"
+
+kubectl apply -Rf "$(dirname "$0")/config-transport-encryption"
+
+go_test_e2e -timeout=1h ./test/e2e_new -run TLS || fail_test
+
 if ! ${LOCAL_DEVELOPMENT}; then
   go_test_e2e -tags=sacura -timeout=40m ./test/e2e/... || fail_test "E2E (sacura) suite failed"
 fi
 
-success
+success
```

Comment:
There is a linter error, let's not change code that is not related to the changes

```
[EOF Newline] reported by reviewdog 🐶
Missing newline

Raw Output:
test/reconciler-tests.sh:61: Missing newline
```

Reply:
@pierDipi  ohh bcz I saw on the main branch, there is no new line in this file, so I removed the new line.